client/pages/recipe.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import FullRecipe from '../components/full-recipe';
+
+export default class Recipe extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      recipe: []
+    };
+  }
+
+  componentDidMount() {
+    const recipe = localStorage.getItem('user-full-recipe');
+    if (!recipe) {
+      return null;
+    }
```

Comment:
Instead of grabbing this from localStorage, can you fetch it from the api?

https://spoonacular.com/food-api/docs#Get-Recipe-Information